### PR TITLE
Turn on --bg option by default in deploy_test_gcp.sh

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Turn on --bg option by default in deploy_test_gcp.sh
- Add --fg option instead

### CODE REVIEW

File: deploy_test_gcp.sh [hunk 1]
```diff
@@ -141,30 +141,18 @@ function deploy_test() {
...
-        deploy_test 2 test_integration_node &
-        deploy_test 3 test_integration_sharding &
-        deploy_test 4 test_integration_blockchain &
-        deploy_test 5 test_integration_consensus &
-        deploy_test 6 test_integration_dapp &
-        deploy_test 7 test_integration_he_protocol &
-        deploy_test 8 test_integration_he_sharding &
-        deploy_test 9 test_integration_event_handler
-    else
+    if [[ $FOREGROUND_OPTION = "--fg" ]] || [[ $STOP_ONLY_OPTION = "--stop-only" ]]; then
```
Comment (kriii) [inline deploy_test_gcp.sh:155]: This logic changed parallelized into serialized when the stop-only option turned on, is it intended? (It seems that all two choices are workable, but I asked if there could be some details in this.)

**Review by kriii — APPROVED** [review]
LGTM with a comment!
Thanks!

File: deploy_test_gcp.sh (hunk 1 above)
Comment (platfowner) [reply deploy_test_gcp.sh]: Ah, --stop-only needs to be removed, and just done. Thanks!

**Comment by platfowner** [comment]
Let me merge this PR now. Feel free to add any further comments.

**Pull request merged**